dogs.html

```diff
@@ -0,0 +1,144 @@
+          borking doggo, puggorino ur givin me a spook noodle horse.
+          <button type="button">Adopt</button>
+        </p>
+  </section>
+  <section class="dog"><img src="adopt-a-dog-spring2019-master/images/violet-card.jpg" alt="dog photo" />
+      <h3 class="profileheader">Violet</h3>
+      <h4 class="costtoadopt">Cost to Adopt:</h4><p class="costtoadoptnumber">$123.45</p>
+        <p> Doggo ipsum dat tungg tho doggorino maximum borkdrive, yapper. Thicc what a nice floof very hand that feed shibe puggo snoot
+          borking doggo, puggorino ur givin me a spook noodle horse.
+          <button type="button">Adopt</button>
+        </p>
+  </section>
+  <section class="dog"><img src="adopt-a-dog-spring2019-master/images/piper-card.jpg" alt="dog photo" />
+      <h3 class="profileheader">Piper</h3>
+      <h4 class="costtoadopt">Cost to Adopt:</h4><p class="costtoadoptnumber">$123.45</p>
+        <p> Doggo ipsum dat tungg tho doggorino maximum borkdrive, yapper. Thicc what a nice floof very hand that feed shibe puggo snoot
+          borking doggo, puggorino ur givin me a spook noodle horse.
+          <button type="button">Adopt</button>
+        </p>
+  </section>
+  <section class="dog"><img src="adopt-a-dog-spring2019-master/images/maximus-card.jpg" alt="dog photo" />
+      <h3 class="profileheader">Maximus</h3>
+      <h4 class="costtoadopt">Cost to Adopt:</h4><p class="costtoadoptnumber">$123.45</p>
+        <p> Doggo ipsum dat tungg tho doggorino maximum borkdrive, yapper. Thicc what a nice floof very hand that feed shibe puggo snoot
+          borking doggo, puggorino ur givin me a spook noodle horse.
+          <button type="button">Adopt</button>
+        </p>
+  </section>
+  <section class="dog"><img src="adopt-a-dog-spring2019-master/images/luna-card.jpg" alt="dog photo" />
+      <h3 class="profileheader">Luna</h3>
+      <h4 class="costtoadopt">Cost to Adopt:</h4><p class="costtoadoptnumber">$123.45</p>
+        <p> Doggo ipsum dat tungg tho doggorino maximum borkdrive, yapper. Thicc what a nice floof very hand that feed shibe puggo snoot
+          borking doggo, puggorino ur givin me a spook noodle horse.
+          <button type="button">Adopt</button>
+        </p>
+  </section>
+  <section class="dog"><img src="adopt-a-dog-spring2019-master/images/stella-card.jpg" alt="dog photo" />
+      <h3 class="profileheader">Stella</h3>
+      <h4 class="costtoadopt">Cost to Adopt:</h4><p class="costtoadoptnumber">$123.45</p>
+        <p> Doggo ipsum dat tungg tho doggorino maximum borkdrive, yapper. Thicc what a nice floof very hand that feed shibe puggo snoot
+          borking doggo, puggorino ur givin me a spook noodle horse.
+          <button type="button">Adopt</button>
+        </p>
+  </section>
+</article>
+
+<button type="button">See More Dogs</button>
+
+<div class="footersection">
```

Comment:
This is kind of unnecessary, the footer tag itself should be enough for a container.